src/main/java/eu/bbmri/eric/csit/service/negotiator/database/model/Organization.java

```diff
@@ -26,7 +30,13 @@
 @NamedEntityGraph(
     name = "organization-with-detailed-resources",
     attributeNodes = {@NamedAttributeNode(value = "resources")})
-public class Organization extends BaseEntity {
+public class Organization {
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "resource_id_seq")
+  @SequenceGenerator(name = "resource_id_seq", initialValue = 10000)
```

Comment:
This is to avoid unique key violations that arise when you migrated data from v2.